src/main/daml/Daml/Finance/Claims/Util/Builders.daml

```diff
@@ -192,3 +193,20 @@ createOptionsClaim dateToTime ownerReceives strike spot cashInstrumentCid expiry
     claim = if ownerReceives then claimAmount else give claimAmount
   in
     prepareAndTagClaims dateToTime [claim] "Option payment"
+
+-- | Create the claim for a physically settled European option.
+createEuropeanPhysicalClaim : (Date -> Time) -> Bool -> Decimal -> Deliverable -> Deliverable ->
+  Date -> Bool -> TaggedClaim
+createEuropeanPhysicalClaim dateToTime ownerReceives strike underlying ccy
+  expiryDate isCall =
+  let
+    exerciseClaim = if isCall then
```

Comment:
this indentation is a bit unusual, not sure it conforms to our style guide

Reply:
I moved the `if` to the next line, to have the two claims aligned and to have standard 2 space indentation.